src/components/LogIn/LogIn.js

```diff
@@ -17,21 +28,25 @@ const LogIn = ({ children }) => {
     }
   });
 
+  const button = !useSelector(isConnecting)
+    ? generateButton("p-button", "_#", "Connecting...")
```

Comment:
What about disabling the button rather then changing the URL to a hashbang

Reply:
You can't disable an anchor tag. I've switched it to use a `<button>` when no `visitURL` is present, and the anchor tag when it is.